demo/source/test.ts

```diff
@@ -274,12 +274,14 @@ it('no trades, sync before/after', async () => {
 })
 
 fit('one trade', async () => {
-	await resetUniswapAndAccount(contracts.uniswapExchange, contracts.token0, contracts.token1, rpcSignerAddress, 1n, 1n)
-	const blockNumber = await rpc.getBlockNumber() // block where liquidity was minted, implicit sync call, price is 1:1 at the end of this block
-	await setPrice(contracts.uniswapExchange, contracts.token0, contracts.token1, 2n, 1n) // one block at 1:1 price, followed by a block with a sync call in it and 2:1 price at end
-	await contracts.uniswapExchange.sync() // block with 2:1 price for full duration and sync included
+	await resetUniswapAndAccount(contracts.uniswapExchange, contracts.token0, contracts.token1, rpcSignerAddress, 5n, 1n)
+	await setPrice(contracts.uniswapExchange, contracts.token0, contracts.token1, 1n, 1n) // First set price to 1:1 and sync. This will take two blocks, and will ensure cumulativeLast is in storage trie
```

Comment:
Recommend changing this to `contracts.uniswapExchange.sync()` along with the above recommended change to have the reset set to `1`.  Alternatively, we could update `resetUniswapAndAccount` so that it does one sync after the mint to ensure we *always* start with a non-zero accumulator.

Reply:
hmmm, sure i can sync instead. i was choosing a non 1:1 to force the block every time, but a sync would do that too